Formula/molten-vk.rb

```diff
@@ -0,0 +1,107 @@
+class MoltenVk < Formula
+  desc "Implementation of the Vulkan graphics and compute API on top of Metal"
+  homepage "https://github.com/KhronosGroup/MoltenVK"
+  url "https://github.com/KhronosGroup/MoltenVK/archive/v1.0.15.tar.gz"
+  sha256 "32b3942722b9feb8250a88d44a018c08f53816f2f895b6199b4b8c2e812d8fc2"
+
+  depends_on "cmake" => :build
+  depends_on :macos => :sierra # requires IOSurface/IOSurfaceRef.h
+
+  # MoltenVK depends on very specific revisions of its dependencies.
+  # For each resource the path to the file describing the expected
+  # revision is listed.
+  resource "cereal" do
+    # ExternalRevisions/cereal_repo_revision
+    url "https://github.com/USCiLab/cereal/archive/51cbda5f30e56c801c07fe3d3aba5d7fb9e6cca4.tar.gz"
+    sha256 "26361b539fe50eee308b564faa2742166d2922a7ab0bd4870ac55708581228c8"
+  end
+
+  resource "glslang" do
+    # ExternalRevisions/glslang_repo_revision
+    url "https://github.com/KhronosGroup/glslang/archive/2c8265bb620300047bb9241e39c60c7afe3a81cb.tar.gz"
+    sha256 "3ffad7f47a609fe0480497344df86191beaa5a4562d6e5f6f26fa251d914d8e5"
+  end
+
+  resource "spirv-cross" do
+    # ExternalRevisions/SPIRV-Cross_repo_revision
+    url "https://github.com/KhronosGroup/SPIRV-Cross/archive/be7425ef70231ab82930331959ab487d605d0482.tar.gz"
+    sha256 "21afa0e32dd2d75ee51c4f5c0d16330a9d80d435d6ac9900d0c13bffdfd7b682"
+  end
+
+  resource "vulkan-headers" do
+    # ExternalRevisions/Vulkan-Headers_repo_revision
+    url "https://github.com/KhronosGroup/Vulkan-Headers/archive/b1577d5fbd5424c863710aa156aaafa77cae3de8.tar.gz"
+    sha256 "626332d056a15ca3951434a170fce63daf378a209dc41038c54381341cb6c0b4"
+  end
+
+  resource "spirv-tools" do
+    # External/glslang/known_good.json
+    url "https://github.com/KhronosGroup/SPIRV-Tools/archive/f2c93c6e124836797311facb8449f9a0b76fefc2.tar.gz"
+    sha256 "d8f4fd9a58ac6a70379ebb9d76e72c6c3b1d928d1e1a6242438d1ae9d39fa80c"
+  end
+
+  resource "spirv-headers" do
+    # External/glslang/known_good.json
+    url "https://github.com/KhronosGroup/SPIRV-Headers/archive/12f8de9f04327336b699b1b80aa390ae7f9ddbf4.tar.gz"
+    sha256 "7d8140b71007d2a7e74903d1b9e66c202c7975d0b3cb60aa1d3af46c9eccfd59"
+  end
+
+  def install
+    (buildpath/"External/cereal").install resource("cereal")
+    (buildpath/"External/glslang").install resource("glslang")
+    (buildpath/"External/glslang/External/spirv-tools").install resource("spirv-tools")
+    ln_s "../extension_enum.inc", buildpath/"External/glslang/External/spirv-tools/source"
```

Comment:
This should use `install_symlink`

Reply:
I think this symlink isn't needed anymore, verifying and removing.